src/main/kotlin/org/jitsi/nlj/rtcp/KeyframeRequester.kt

```diff
@@ -209,3 +212,5 @@ private fun PacketInfo.getPliOrFirPacket(): RtcpFbPacket? {
         else -> null
     }
 }
+
+private val NEVER = Instant.MIN
```

Comment:
Should this go in the companion object?

Reply:
really it should probably go in a time utils file, as i think i've defined this same thing somewhere else already.